tutor/templates/config/defaults.yml

```diff
@@ -2,6 +2,8 @@
 # This file includes all Tutor setting defaults. Settings that do not have a
 # default value, such as passwords, should be stored in base.yml.
 # This must be defined early
+ATLAS_BRANCH: "main"
```

Comment:
Should be "ATLAS_REVISION" and it should be "main" only in nightly, based on the value of "OPENEDX_COMMON_VERSION".